lib/defaults.js

```diff
@@ -19,5 +19,9 @@ module.exports = {
   labelDraft: {
     name: "PR: draft",
     color: "6a737d"
+  },
+  labelPartiallyApproved: {
+    name: "PR: partially-approved",
+    color: "7E9C82"
```

Comment:
I like the color!

Reply:
I just took what was for review-required and tweaked it a little bit 😄